frontend/src/features/contacts/contact/detail/utils.ts

```diff
@@ -0,0 +1,89 @@
+import { IContactAddress } from 'interfaces/IContact';
+
+export const sortAddresses = (a1: IContactAddress, a2: IContactAddress) => {
+  if (a2.addressType.id === a1.addressType.id) {
+    return 0;
+  }
+
+  if (a2.addressType.id === 'MAILING') {
```

Comment:
nit: I prefer we use enums/constants here. I believe we have that in the frontend already